packages/components/src/contactDetailsForm/ContactDetailsForm.js

```diff
@@ -13,15 +13,23 @@ import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import HeaderWithIcon from '../headerWithIcon/HeaderWithIcon';
 
 type Props = {|
-  onChangeEmail: string => void,
-  onChangePhoneNumber: string => void,
-  onChangeCountryCode: (?string) => void,
+  +disableEmail: ?boolean,
```

Comment:
`disableEmail` cannot be `null` as `disabled` on TextInput cannot be null (but it's optional) and also we are generating it with `Boolean(this.props.userId)` which can never be null. So maybe change that type to be `+disableEmail?: boolean` and set a default disabled if the prop is not there, or even `+disableEmail: boolean`

Reply:
good point 👍

Reply:
done